lib/slack-ruby-bot/commands/base.rb

```diff
@@ -81,6 +85,11 @@ def scan(match, &block)
           self.routes[match] = { match_method: :scan, block: block }
         end
 
+        def attachment(match, &block)
+          self.routes ||= ActiveSupport::OrderedHash.new
```

Comment:
Sounds like we need to make `self.routes` a method that does `@routes ||=` or something like that?

Reply:
Agree.